### PR TITLE
IGAPP-1224: Fix search result highlights

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: api-client/src/utils/getExcerpt.ts [hunk 1]
```diff
@@ -37,27 +37,34 @@ type ExcerptOptions = {
 }
 
 const getExcerpt = (text: string, { query, maxChars, replaceLineBreaks }: ExcerptOptions): string => {
-  const match = query ? normalizeString(text).match(normalizeString(query)) : []
```
Comment (sarahsporck) [inline api-client/src/utils/getExcerpt.ts:40]: Before it tried to also match regex strings, meaning writing `?` would break the search

Comment (LeandraH) [reply api-client/src/utils/getExcerpt.ts:40]: Ouch, good catch!

**Review by LeandraH — APPROVED** [review]
Very nice, tested in Firefox and iOS emulator

**Comment by f1sh1918** [comment]
Nice find! Just a small question to "Umlaute"
Should this match?
image

**Comment by sarahsporck** [comment]
> Also "u" matches "ü" i know this is not related to your pr. Just a general question :)

Jep. This should match. That's the reason why we normalize all our strings in the search.

**Review by f1sh1918 — APPROVED** [review]

**Pull request merged**